pyproject.toml

```diff
@@ -26,6 +26,8 @@ numpy = "^1.23.1"
 pandas = "^1.4.3"
 scipy = "^1.9.0"
 requests = "^2.28.1"
+tqdm = "^4.64.1"
+httpx = "^0.23.0"
```

Comment:
side note: is there a good way for us to put these dependencies not in core library but just for resolver or we can't really do that since resolver is currently under core lib?
cc @impreso

Reply:
So you can either use the dev dependencies or extras. So with extras you would do something like `pip install open-rarity[opensea]` to get the libraries that are associated with that portion of the library. I have only briefly used this pattern in the past.

Documentation [here](https://python-poetry.org/docs/pyproject/#extras)

Reply:
oh this is cool - perhaps we do that

Reply:
Punted to a new issue.